Source/Core/Core/HLE/HLE_OS.cpp

```diff
@@ -204,76 +207,364 @@ void HLE_LogVFPrint(const Core::CPUThreadGuard& guard)
   HLE_LogFPrint(guard, ParameterType::VariableArgumentList);
 }
 
+namespace
+{
+class HLEPrintArgsVAList final : public HLEPrintArgs
+{
+public:
+  HLEPrintArgsVAList(const Core::CPUThreadGuard& guard, HLE::SystemVABI::VAList& va_list)
+      : m_guard(guard), m_va_list(va_list)
+  {
```

Comment:
Shouldn't `HLE::SystemVABI::VAList& va_list` be passed as a pointer in case it's modified according to Dolphin's coding style? If the coding style has changed or allows some exceptions, they might be worth mentioning in the [Contributing.md](https://github.com/dolphin-emu/dolphin/blob/master/Contributing.md) file.

Reply:
Mhm, I think that's actually ambiguous, since we're not actually assigning a new value to the passed `va_list`, but yeah sure it doesn't hurt.